app/views/apps/new.html.erb

```diff
@@ -0,0 +1,44 @@
+<div class="content">
+
+  <h1 class="header">New Adoption Application</h1>
+
+  <section id="new-app" class="bold">
+    <%= form_tag "/apps" do %>
+
+  <% @favorite_pets.each do |pet| %>
+  <section id="<%= pet.id %>-info" class="pet-list">
+
+      <img class="pet-image-small" src="<%= pet.image %>" alt="<%= pet.name %> image"> <p>
+      <h2><a id="<%= pet.name %>"class="pet-link hover" href="/pets/<%= pet.id %>"><%= pet.name %></a></h2>
+      <br>
+
+  </section>
+  <br>
+
+  <% end %>
+      <%= label_tag :pets %><br>
+      <%= select_tag "pet_ids[]", options_for_select(@favorite_pets.map {|pet| [pet.name, pet.id]}), multiple: :true, required: :true %><br><br>
```

Comment:
proud of this. If you put the tag in "" and add a [] it can populate params as an array. Now when its sent to the controller we have an array of pet.ids

Reply:
oh ok thats sick!